hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -1471,7 +1471,7 @@
         <xs:complexType>
             <xs:complexContent>
                 <xs:extension base="hazelcast-bean">
-                    <xs:all>
+                    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
actually it works just fine. the rules are quite complicated, see this: http://stackoverflow.com/questions/5214555/understanding-xsdchoice-and-minoccurs